src/MultiSelect/Menu.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import propTypes from 'prop-types'
+import { multiSelectedPropType } from '../common-prop-types.js'
+import { Empty } from '../Select/Empty.js'
+
+const createHandler = ({ active, onChange, selected, value, label }) => e => {
+    e.stopPropagation()
+    e.preventDefault()
+
+    // If the option is active, remove it from selected
+    if (active) {
+        const filtered = selected.filter(selection => {
+            const matchesLabel = label === selection.label
+            const matchesValue = value === selection.value
+            return !matchesLabel && !matchesValue
+        })
+        return onChange(filtered)
+    }
```

Comment:
I find the choice of words here very hard to understand. It took me a while to understand that `active` actually means that the option is selected and `selected` contains all selected options.
Can we change the prop name? Maybe `selected`/`isSelected` and `selectedOptions`

Reply:
I understand. I've already refactored it a bit: 2f5ea2f9976b1cf1725e6b2bc9a485ba1723791e, let me know if that helps, or if you'd prefer changing the prop names.

Reply:
Yes, it's much better